modules/settings/variables.tf

```
@@ -64,57 +65,51 @@ variable "redis_pass" {
 }
 
 variable "redis_enable_non_ssl_port" {
-  default     = true
+  default     = null
   type        = bool
   description = "If true, the external Redis instance will use port 6379, otherwise 6380"
 }
 
 variable "redis_use_password_auth" {
-  default     = false
+  default     = null
   type        = bool
   description = "Redis service requires a password."
 }
 
 variable "redis_use_tls" {
-  default     = false
+  default     = null
   type        = bool
   description = "Redis service requires TLS"
 }
 
 # Azure
 # -----
-variable "user_data_azure_container_name" {
+variable "azure_container_name" {
+  default     = null
   type        = string
   description = "Azure storage container name"
 }
 
-variable "user_data_azure_account_key" {
+variable "azure_account_key" {
+  default     = null
   type        = string
   description = "Azure storage account key"
 }
 
-variable "user_data_azure_account_name" {
+variable "azure_account_name" {
+  default     = null
   type        = string
   description = "Azure storage account name"
 }
 
 # TFE
 # ---
 variable "release_sequence" {
+  default     = null
   type        = string
```

I believe this should be a number so that the settings file accurately renders this to a JSON number just like [Replicated expects](https://help.replicated.com/docs/native/customer-installations/automating/).

```suggestion
  type        = number
```